<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>ViewBook</class>
 <widget class="QWidget" name="ViewBook">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>1120</width>
    <height>619</height>
   </rect>
  </property>
  <property name="minimumSize">
   <size>
    <width>780</width>
    <height>0</height>
   </size>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <widget class="QLabel" name="coverLabel">
   <property name="geometry">
    <rect>
     <x>250</x>
     <y>30</y>
     <width>200</width>
     <height>320</height>
    </rect>
   </property>
   <property name="text">
    <string/>
   </property>
  </widget>
  <widget class="QPushButton" name="prebookButton">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>390</y>
     <width>150</width>
     <height>40</height>
    </rect>
   </property>
   <property name="text">
    <string>Reserve</string>
   </property>
  </widget>
  <widget class="QLabel" name="titleLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>20</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>TITLE</string>
   </property>
  </widget>
  <widget class="QLabel" name="authorLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>70</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>AUTHOR</string>
   </property>
  </widget>
  <widget class="QLabel" name="isbnLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>120</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>ISBN</string>
   </property>
  </widget>
  <widget class="QLabel" name="pgLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>170</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>PAGE COUNT</string>
   </property>
  </widget>
  <widget class="QLabel" name="availableLabel">
   <property name="geometry">
    <rect>
     <x>280</x>
     <y>390</y>
     <width>140</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>140</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>Available</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignCenter</set>
   </property>
  </widget>
  <widget class="QLabel" name="genreLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>220</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>GENRE</string>
   </property>
  </widget>
  <widget class="QLabel" name="ddLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>270</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>DD</string>
   </property>
  </widget>
  <widget class="QLabel" name="blurbLabel">
   <property name="geometry">
    <rect>
     <x>210</x>
     <y>460</y>
     <width>691</width>
     <height>131</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>100</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>750</width>
     <height>150</height>
    </size>
   </property>
   <property name="text">
    <string>BLURB</string>
   </property>
   <property name="alignment">
    <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignTop</set>
   </property>
  </widget>
  <widget class="QLabel" name="publishdateLabel">
   <property name="geometry">
    <rect>
     <x>510</x>
     <y>320</y>
     <width>500</width>
     <height>40</height>
    </rect>
   </property>
   <property name="minimumSize">
    <size>
     <width>100</width>
     <height>40</height>
    </size>
   </property>
   <property name="maximumSize">
    <size>
     <width>500</width>
     <height>40</height>
    </size>
   </property>
   <property name="text">
    <string>PUBLISH</string>
   </property>
  </widget>
 </widget>
 <resources/>
 <connections/>
</ui>
